examples/advanced_usage.py

```
@@ -0,0 +1,196 @@
+        result = analyze_single_image(
+            geospy,
+            image_path,
+            context="Historic tower in an urban setting",
+            guess="Turkey"
+        )
+        
+        if "error" not in result:
+            # Display results
+            print("\nAnalysis Results:")
+            print(f"Interpretation: {result.get('interpretation', 'N/A')[:100]}...")
+            print(f"\nTop Locations:")
```

Ruff (0.14.0)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)